src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,35 @@
+.contacts {
+  &__title {
+    @extend %text-h2;
+    text-transform: uppercase;
+    margin-bottom: 10px;
+  
+    @include Desktop {
+      margin-bottom: 30px;
+    }
+  }
+
+  &__paragraph {
+    font-size: 14px;
+    margin-bottom: 30px;
+
+    @include Tablet {
+      font-size: 16px;
+      margin-bottom: 20px;
+    }
+
+    @include Desktop {
+      margin-bottom: 40px;
+    }
+  }
+
+  &__image {
+    @extend %bacground-images-settings;
```

Comment:
There is a typo in the extend name. It should be '%background-images-settings' instead of '%bacground-images-settings'.